tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/fsm/FSMTrainingVectorSource.scala

```diff
@@ -34,7 +36,7 @@ abstract class FSMTrainingVectorSource(
     if (!vectorIter.hasNext) {
       resultSoFar
     } else {
-      taskHelper(resultSoFar + vectorIter.next.task, vectorIter)
+      taskHelper(resultSoFar + vectorIter.next().task, vectorIter)
     }
```

Comment:
I was just being safe. I was not sure exactly how .toSet called on an iterator would be internally implemented.